src/main/client/src/common/requestButton/buttonCollectStates.js

```diff
@@ -0,0 +1,35 @@
+const buttonCollectStates = (props, book) => {
+  let request,
+    colour,
+    buttonText,
+    popupText,
+    basic = false;
+
+  switch (book.collectState) {
+    case "Collector - Collection Started":
+      request = props.collectBook;
+      colour = "green";
+      buttonText = "Confirm";
+      popupText = "Are you sure?";
+      basic = true;
+      break;
```

Comment:
There may not need to be a confirmation for collecting, we should ask Rob the PO Bro